CommonTools/TrackerMap/src/TrackerMap.cc

```diff
@@ -3623,21 +3623,23 @@ saveAsSingleLayer=false;
 TLegend* TrackerMap::buildLegend() {
   if ( legInfos_.empty() ) {
     legInfos_.reserve(8);
-    legInfos_.push_back(new TH1F("green","green",1,0.,1.));
+    Double_t x[4] = {.2,.7,.6,.2};
+    Double_t y[4] = {.5,.1,.9,.5};
```

Comment:
does this have any significance and you intend to display these polylines in these coordinates? (the histograms were all unfilled in the last iteration).
Empty TPolyLine can be used if 4 x-y points are not meaningful.

Reply:
hi @slava77 - is there much harm? (aside from 8 additional doubles in a macro)

Reply:
since a much more complicated solution is introduced, it sounds like it's needed.
But in that case, why not 42 points or some other important number.

Reply:
@slava77 These points are not meaningful. I just removed them